index.js

```diff
@@ -118,6 +140,12 @@ function walker(node, parameters, outerFont) {
           value: node.attribs.color,
           range: [offset, length]
         });
+      } else if (customMatcher !== null) {
```

Comment:
In your case having the custom matcher as last option works fine, but others might want to override builtin behaviour. So it would better come first and then the matcher should return a boolean to indicate if we should continue with builtin matchers.